core/src/main/java/org/elasticsearch/bootstrap/Elasticsearch.java

```diff
@@ -53,4 +118,97 @@ public static void main(String[] args) throws StartupError {
     static void close(String[] args) throws IOException {
         Bootstrap.stop();
     }
+
+    @Argument(handler = SubCommandHandler.class, metaVar = "command")
+    @SubCommands({
+        @SubCommand(name = "start", impl = StartCommand.class),
+        @SubCommand(name = "version", impl = VersionCommand.class)
+    })
+    Command command;
+
+    static abstract class Command {
+
+        @Option(name = "-h", aliases = "--help", usage = "print this message")
```

Comment:
@dakrone Is there really a convention here? `ant` doesn't support `--version`, for example. I'm not sure I see the need for it, and from a technical perspective I just don't want to maintain two ways to do the same thing.